src/test/java/cms/sre/emailnotifier/dao/TestMimeMessageSMTPDao.java

```diff
@@ -0,0 +1,35 @@
+package cms.sre.emailnotifier.dao;
+
+import cms.sre.emailnotifier.model.Email;
+import cms.sre.emailnotifier.service.EmailService;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.Properties;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class TestMimeMessageSMTPDao {
```

Comment:
You should test to make sure an instance of SMTPDao is autowired, and that instance is of type MimeMessageSMTPDao